clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/IsolatedIngestionBackend.java

```diff
@@ -259,12 +271,15 @@ public void completed(
           long offset,
           String message,
           Optional<LeaderFollowerStateType> leaderState) {
+        VeniceStoreVersionConfig config = getConfigLoader().getStoreConfig(kafkaTopic);
+        config.setRestoreDataPartitions(false);
+        config.setRestoreMetadataPartition(false);
```

Comment:
Why move this before the if? Is it significant? I see the config is only used inside of the if, which makes this refactoring a bit weird...

Reply:
Yeah either way should work, I think I was battling with the coverage but this place is not the critical part actually, I will move it back.